salt/states/ssh_auth.py

```diff
@@ -329,7 +329,7 @@ def present(
                 keyline = keyline.split(' ')
                 key_type = keyline[0]
                 key_value = keyline[1]
-                key_comment = keyline[2] if len(keyline) > 2 else ''
+                key_comment = ' '.join(keyline[2:]) if len(keyline) > 2 else ''
```

Comment:
Could be simply replaced with:
```python
key_comment = ' '.join(keyline[2:])
```
The result is an empty string in case `keyline` has less than 3 items.

Reply:
We should also add some tests here. Presumably that shouldn't be too difficult for this scenario...